Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/CalendarDatePickerSamplePage.xaml

```diff
@@ -0,0 +1,79 @@
+﻿<Page x:Class="Uno.Gallery.Views.SamplePages.CalendarDatePickerSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.MaterialTemplate>
+				<DataTemplate>
+					<StackPanel Spacing="20">
+						<smtx:XamlDisplay UniqueKey="CalendarViewSamplePage_Material_CalendarDatePicker"
+										  smtx:XamlDisplayExtensions.Header="Default">
+							<CalendarDatePicker Header="Date"
+												x:Name="Material_CalendarDatePicker"
+												AutomationProperties.AutomationId="Material_CalendarDatePicker"
+												Style="{StaticResource MaterialCalendarDatePickerStyle}" />
+						</smtx:XamlDisplay>
+						<smtx:XamlDisplay UniqueKey="CalendarViewSamplePage_Material_CalendarDatePicker_Disabled"
+										  smtx:XamlDisplayExtensions.Header="Disabled">
+							<CalendarDatePicker Header="Date"
+												IsEnabled="False"
+												x:Name="Material_CalendarDatePicker_Disabled"
+												AutomationProperties.AutomationId="Material_CalendarDatePicker_Disabled"
```

Comment:
dont need both x:Name+AutoProp.AutoId, just x:Name is enough
the value of x:Name will take precedence for `Marked(key)` should they be different

Reply:
UI Testing doesn't seem to work unless I have both